vademecum_VT.md

```diff
@@ -47,13 +54,19 @@ Dalla versione Quantum, Firefox integra il nuovo sistema Style che velocizza il
 Questa tecnologia è parte del nuovo browser engine Servo ed è presente anche sulle versioni per dispositivi mobile, ovvero su Android e iOS.  
 
 #### WebVR |QRcode|
-Esplora la realtà virtuale direttamente da Firefox, senza necessità di installare software specifici. Con WebVR la realtà aumentata e virtuale è gratuita, semplice e per tutti.
+Firefox supporta lo standard _WebVR_, che permette di avere la realtà virtuale direttamente sul browser in tutte le piattaforme.
+Mozilla ha sviluppato il framework open source _A-Frame_ che, utizzando i web component e three.js, permette di scrivere le scene 3D direttamente in html. In questo modo chiunque può creare un ambiente 3D, mettere un video 3D. Inoltre A-Frame ha diversi sottoprogetti come A-Terrain, per visualizzare dati geografici, A-Blast, per creare giochi con WebVr, A-Painter, per disegnare in ambiente 3D.
 
 #### Rust |QRcode|
-Rust è un linguaggio compilato con performance simili a quelle del C e garantisce, in aggiunta, la sicurezza del programma. È sviluppato da Mozilla ed è un componente fondamentale in Firefox Quantum.
+Rust è un linguaggio,realizzato e mantenuto da Mozilla, compilato ad elevate performance, in aggiunta, il compilatore garantisce la sicurezza del programma.
```

Comment:
Non vedo perchè accorciare la frase... inoltre è poco chiara "compilato a<s>d</s> elevate performance, in aggiunta, il compilatore garantisce la sicurezza del programma"... cosa significa? riporto all'originale